auth_fetch_key_helper.go

```diff
@@ -0,0 +1,118 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+	"path/filepath"
+	"regexp"
+
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-io/go-utils/pathutil"
+)
+
+func fetchPrivateKey(apiKeyPath string) ([]byte, string, error) {
+	// see these in the altool's man page
+	var keyPaths = []string{
```

Comment:
i am not sure if it worth doing this.

this logic checks if the same (same file name) API key already exists locally, if so, we use the local key, which can be an issue, if the user provided one, has the same filename, but different content

Reply:
Removed but still using ~/.appstoreconnect path to store file, this is not really needed (only for Application Loader Step) in that case a specific logic in the step (not package) to copy the key would make more sense.